h/static/scripts/widget-controller.js

```diff
@@ -5,6 +5,21 @@ var angular = require('angular');
 var events = require('./events');
 var SearchClient = require('./search-client');
 
+/**
+ * Returns the group ID of the first annotation in @p results whose
```

Comment:
This is me thinking of Doxygen where `@p` is a way of referencing a parameter in the description of a method. I'll look up the correct way to do this in JSDoc.